readme.md

```diff
@@ -99,37 +112,70 @@ import Preferences
 
 @NSApplicationMain
 final class AppDelegate: NSObject, NSApplicationDelegate {
-	@IBOutlet private var window: NSWindow!
+    @IBOutlet private var window: NSWindow!
 
-	let preferencesWindowController = PreferencesWindowController(
-		viewControllers: [
-			GeneralPreferenceViewController(),
-			AdvancedPreferenceViewController()
-		]
-	)
+    let preferencesWindowController = PreferencesWindowController(
+        preferences: [
+            GeneralPreferenceViewController(),
+            AdvancedPreferenceViewController()
+        ]
+    )
 
-	func applicationDidFinishLaunching(_ notification: Notification) {}
+    func applicationDidFinishLaunching(_ notification: Notification) {}
 
-	@IBAction
-	func preferencesMenuItemActionHandler(_ sender: NSMenuItem) {
-		preferencesWindowController.showWindow()
-	}
+    @IBAction
+    func preferencesMenuItemActionHandler(_ sender: NSMenuItem) {
+        preferencesWindowController.showWindow()
+    }
 }
 ```
 
+### Preferences Tab Styles
+
+When you create the `PreferencesWindowController`, you can also switch between the `NSToolbarItem`-based style (default) and the `NSSegmentedControl`:
+
+```swift
+    // ...
+    let preferencesWindowController = PreferencesWindowController(
+        preferences: [
+            GeneralPreferenceViewController(),
+            AdvancedPreferenceViewController()
+        ],
+        style: .segmentedControl
+    )
+    // ...
+```
+
+`.toolbarItem` style:
+
+![NSToolbarItem based (default)](images/toolbar-item.png)
+
+`.segmentedControl` style:
+
+![NSSegmentedControl based](images/segmented-control.png)
 
 ## API
 
 ```swift
-protocol Preferenceable: AnyObject {
-	var toolbarItemTitle: String { get }
-	var toolbarItemIcon: NSImage { get }
+public protocol PreferencePane: AnyObject {
+    var preferencePaneIdentifier: PreferencePaneIdentifier { get }
+    var toolbarItemTitle: String { get }
+    // Defaults to an empty image
+    var toolbarItemIcon: NSImage { get }
+    var viewController: NSViewController { get }
+}
+
+public enum PreferencesStyle {
+    case toolbarItems
+    case segmentedControl
 }
 
 class PreferencesWindowController: NSWindowController {
-	init(viewControllers: [Preferenceable])
-	func showWindow()
-	func hideWindow()
+    init(preferencePanes: [PreferencePane], 
+         style: PreferencesStyle = .toolbarItems, 
+         animated: Bool = true)
+    func show(preferencePane: PreferencePaneIdentifier? = nil)
+    func hideWindow()
```

Comment:
I just noticed we have this. It's a bit inconsistent with `.show()` and `.hideWindow()`. Maybe we should add a `.hide()` method too?

Reply:
Yeah, I stumbled upon that, too. AppKit uses showWindow/close by default, and we now have our show/hideWindow in addition to that. Why not just add the `show(preferencePane:)` variant in addition to `showWindow(_:)`, and stick to `close`?

Reply:
So just remove `hideWindow()` and document `close()`? If so, I agree. I think it's also worth documenting `close()` just for convenience of the user.

Reply:
Ok, see #21